CHANGELOG.md

```diff
@@ -28,7 +28,7 @@ The changes are relative to the previous release, unless the baseline is specifi
   decoder->image->gainMap != NULL instead.
 * Write an empty HandlerBox name field instead of "libavif" (saves 7 bytes).
 * Update aom.cmd/LocalAom.cmake: v3.11.0
-* Update avm.cmd: research-v8.0.0
+* Update avm.cmd: research-v8.1.0
```

Comment:
@y-guyon 

In src/codec_avm.c, we have a TODO comment:

```
        // avm does not handle monochrome as of research-v8.0.0.
        // TODO(yguyon): Enable when fixed upstream
        codec->internal->monochromeEnabled = AVIF_FALSE;
```

Do you know if this bug has been fixed in research-8.1.0? Did we file an issue on this bug?

Reply:
> Do you know if this bug has been fixed in research-8.1.0?

It is not fixed in `research-8.1.0`.  
@vrabaud Maybe update the comment in `src/codec_avm.c` to 8.1.0?

> Did we file an issue on this bug?

https://gitlab.com/AOMediaCodec/avm/-/issues/522

Reply:
Done

Reply:
Thank you.

Tentative fix: https://gitlab.com/AOMediaCodec/avm/-/merge_requests/1396